Week1/homework/step-5.js

```diff
@@ -0,0 +1,52 @@
+const mysql = require('mysql');
+const con = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'newWorld',
+});
+
+var Request = require('request');
+
+Request.get('https://restcountries.eu/rest/v2/all', (error, response, body) => {
```

Comment:
very good, you did extra, and get data about all countries from an endpoint. basically combined requests and databases